### PR TITLE
Support testing container in CVP pipeline

### PR DESCRIPTION
**Pull request by phracek**
This pull request allows for testing s2i-python-container in CVP pipeline.

### CODE REVIEW

File: test/run-openshift-remote-cluster [hunk 1]
```diff
@@ -20,18 +20,27 @@ test_python_imagestream
...
 ct_os_check_login || exit 1
 
+# Pull image before going throw tests
+# Exit in case of failure
+if [ "${CVP}" -eq "0" ]; then
+  ct_pull_image "quay.io/centos7/postgresql-10-centos7" "true"
+else
+  oc import-image "postgresql-10-centos7:latest" --from="quay.io/centos7/postgresql-10-centos7:latest" --insecure=true --confirm
+  oc tag "postgresql-10-centos7:latest" "postgresql:10"
+fi
```
Comment (zmiklank) [inline test/run-openshift-remote-cluster:40]: What if this is moved to `test-lib-python.sh` as a function?

**Review by zmiklank — APPROVED** [review]
LGTM. Thanks.

**Comment by phracek** [comment]
[test-all]

**Comment by phracek** [comment]
[test-all]

**Comment by phracek** [comment]
@frenzymadness Can you please have a look at this pull request, especially `RHEL8 - 3.9-minimal`

File: test/run-openshift-remote-cluster [hunk 2]
```diff
@@ -32,6 +28,10 @@ ct_os_check_compulsory_vars
 
 ct_os_check_login || exit 1
 
+ct_os_tag_image_for_cvp "python"
+
+ct_pull_or_import_postgresql
```
Comment (zmiklank) [inline test/run-openshift-remote-cluster:33]: I know I already gave LGTM, but I suppose we should add something like this here.
```suggestion
ct_pull_or_import_postgresql || exit 1
```

File: test/test-lib-python.sh [hunk 3]
```diff
@@ -11,10 +11,25 @@ THISDIR=$(dirname ${BASH_SOURCE[0]})
 source "${THISDIR}/test-lib.sh"
 source "${THISDIR}/test-lib-openshift.sh"
 
+function ct_pull_or_import_postgresql() {
+  # Variable CVP is set by CVP pipeline
+  if [ "${CVP:-0}" -eq "0" ]; then
+    # In case of container or OpenShift 4 tests
+    # Pull image before going throw tests
```
Comment (frenzymadness) [inline test/test-lib-python.sh:18]: Maybe?
```suggestion
    # Pull image before going through tests
```

**Review by frenzymadness — COMMENTED** [review]
I trust your expertise!

**Comment by phracek** [comment]
[test-all]

**Pull request merged**